--- ../src-base/minecraft/net/minecraft/server/MinecraftServer.java
+++ ../src-work/minecraft/net/minecraft/server/MinecraftServer.java
@@ -719,6 +719,8 @@
 
     public static void main(String[] p_main_0_)
     {
+    	//Dimension Init Call
+        solar.dimensions.Dimensions.init();
         Bootstrap.func_151354_b();
 
         try
@@ -933,7 +935,8 @@
 
     public String getServerModName()
     {
-        return "vanilla";
+    	//Change server branding to Dimensions
+        return solar.dimensions.Dimensions.getInstance().getServer().getName();
     }
 
     public CrashReport func_71230_b(CrashReport p_71230_1_)
